resources/testdriver-common-input.js

```diff
@@ -0,0 +1,112 @@
+// This file contains the commonly used functions in pointerevent tests.
```

Comment:
@jgraham @jugglinmike @foolip how do we feel about this being in resources/?

Reply:
I guess common/ is the other place you'd suggest?